opm/parser/eclipse/Units/ConversionFactors.hpp

```diff
@@ -133,6 +133,7 @@ namespace Opm {
             /// \name Force
             /// @{
             constexpr const double Newton = kilogram*meter / square(second); // == 1
+            constexpr const double dyn    = 1e-5*Newton; // cf. wikipedia
```

Comment:
well, it's just the hard, cold truth. but if you prefer, I'll remove the comment.